### ISSUE TITLE
Friday authentication problems / issues / slowness

### ISSUE THREAD

**Issue by FredrikKarlssonSpeech** [humlab-speech/visible-speech-deployment#172]
Today, authentication was slow and prone to not working. Sometimes. Reloading the page sometimes works, if you try more than once.

Difficult to investigate, I know, but once there was a direct error message which could be captured.

![Skärmavbild 2022-12-16 kl  10 30 50](https://user-images.githubusercontent.com/200246/208069258-369d5909-8b86-4065-aa7f-b80a76bc6e8d.png)

**Comment by johanvonboer** [comment]
This could be separate issues. I'm wondering if the first one has to do with some sort of university central DNS-problems they had this morning. But the second issue is more probably related to the sparse-cloning, which was probably rushed out too soon. I found a bug in the session-manager which causes a crash there in some cases, I have fixed this and updated the server. However, I also found a problem in the container-agent which makes sparse cloning default when it shouldn't be.
I have fixed that too, but applying it requires rebuilding the session images, which can take 1+ hour (I think) for the jupyter container. It's currently running.
